### PR TITLE
feat: update nspr

### PR DESCRIPTION
**Pull request by Komorebi-yaodong**
This library provides platform independent non-GUI operating system

Log: nspr would be updated to version 4.6.0.

Issue: https://github.com/deepin-community/sig-deepin-sysdev-team/issues/153

### CODE REVIEW

File: debian/patches/Add-sunway-support.patch [hunk 1]
```diff
@@ -1,258 +0,0 @@
-Description: add sunway support
```
Comment (xzl01) [inline debian/patches/Add-sunway-support.patch]: patch需要更新后保留

**Comment by Zeno-sole** [comment]
其他文件怎么都没了？

**Comment by Zeno-sole** [comment]
删除了很多不该删的文件

**Pull request merged**